Provide better storage path for minio example file

Under `examples/minio/00-minio-deployment.yaml`, minio is configured to use storage from the host path `/tmp/minio`

I am trying out ark on minikube -

```console
$ minikube version
minikube version: v0.25.0

$ kubectl version
Client Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.0", GitCommit:"925c127ec6b946659ad0fd596fa959be43f0cc05", GitTreeState:"clean", BuildDate:"2017-12-15T21:07:38Z", GoVersion:"go1.9.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"", Minor:"", GitVersion:"v1.9.0", GitCommit:"925c127ec6b946659ad0fd596fa959be43f0cc05", GitTreeState:"clean", BuildDate:"2018-01-26T19:04:38Z", GoVersion:"go1.9.1", Compiler:"gc", Platform:"linux/amd64"}
```

In minikube, `/tmp/` does not have enough space for minio to run (minio requires 1Gi) to start up
```console
$ df -hT 
Filesystem     Type      Size  Used Avail Use% Mounted on
devtmpfs       devtmpfs  932M     0  932M   0% /dev
tmpfs          tmpfs    1001M     0 1001M   0% /dev/shm
tmpfs          tmpfs    1001M   17M  984M   2% /run
tmpfs          tmpfs    1001M     0 1001M   0% /sys/fs/cgroup
tmpfs          tmpfs    1001M  4.0K 1001M   1% /tmp
/dev/sda1      ext4       17G  2.0G   14G  13% /mnt/sda1
/hosthome      vboxsf    197G  167G   31G  85% /hosthome
```

This lack of space makes minio fail -
```console
$ kubectl get pods -n heptio-ark-server
NAME                     READY     STATUS             RESTARTS   AGE
ark-57ffbc8947-2n299     1/1       Running            0          48m
minio-7468675c85-9txd9   0/1       CrashLoopBackOff   6          9m
minio-setup-mtmk5        0/1       CrashLoopBackOff   11         48m

$ kubectl logs minio-7468675c85-9txd9
Created minio configuration file successfully at /root/.minio


Trace: 1: /q/.q/sources/gopath/src/github.com/minio/minio/cmd/server-main.go:246:cmd.serverMain()
       2: /q/.q/sources/gopath/src/github.com/minio/minio/vendor/github.com/minio/cli/app.go:499:cli.HandleAction()
       3: /q/.q/sources/gopath/src/github.com/minio/minio/vendor/github.com/minio/cli/command.go:214:cli.Command.Run()
       4: /q/.q/sources/gopath/src/github.com/minio/minio/vendor/github.com/minio/cli/app.go:260:cli.(*App).Run()
       5: /q/.q/sources/gopath/src/github.com/minio/minio/cmd/main.go:155:cmd.Main()
       6: /q/.q/sources/gopath/src/github.com/minio/minio/main.go:71:main.main()
[2018-03-15T12:39:22.402063415Z] [ERROR] Initializing object layer failed (disk path full)
```

Odds are that a new user will try out ark on minikube, and we can have better hostPath that the current configuration -

```yaml
    spec:
      volumes:
      - name: storage
        hostPath:
          path: /tmp/minio
```

--- a/examples/minio/00-minio-deployment.yaml
+++ b/examples/minio/00-minio-deployment.yaml
@@ -30,8 +30,7 @@ spec:
     spec:
       volumes:
       - name: storage
-        hostPath:
-          path: /tmp/minio
+        emptyDir: {}
       containers:
       - name: minio
         image: minio/minio:latest

